recipes/duckx/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, copy, get, rmdir
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+class DuckxConan(ConanFile):
+    name = "duckx"
+    description = " C++ library for creating and updating Microsoft Word (.docx) files."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/amiremohamadi/DuckX/"
+    topics = ("docx", "docx-files", "office")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": False,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+        self.options["pugixml"].header_only = True
```

Comment:
Let me double-check, but I think the better approach in CCI is to set
```
default_options = {
    "pugixml/*:header_only": True
}
```

and have a `validate()` check to ensure that the option is properly set

Reply:
@RubenRBS 
Thank you for your review.
I update this PR. Please review it again.